[2025-05-28T10:45:03.812+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_etl.load_weather_task scheduled__2025-05-28T10:40:00+00:00 [queued]>
[2025-05-28T10:45:03.819+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_etl.load_weather_task scheduled__2025-05-28T10:40:00+00:00 [queued]>
[2025-05-28T10:45:03.820+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-28T10:45:03.834+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_weather_task> on 2025-05-28 10:40:00+00:00
[2025-05-28T10:45:03.839+0000] {standard_task_runner.py:60} INFO - Started process 1200 to run task
[2025-05-28T10:45:03.842+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_data_etl', 'load_weather_task', 'scheduled__2025-05-28T10:40:00+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/weather.py', '--cfg-path', '/tmp/tmpce3esq8e']
[2025-05-28T10:45:03.843+0000] {standard_task_runner.py:88} INFO - Job 51: Subtask load_weather_task
[2025-05-28T10:45:03.887+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_data_etl.load_weather_task scheduled__2025-05-28T10:40:00+00:00 [running]> on host f1bd559e8a8b
[2025-05-28T10:45:03.960+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_etl' AIRFLOW_CTX_TASK_ID='load_weather_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-28T10:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-28T10:40:00+00:00'
[2025-05-28T10:45:04.402+0000] {python.py:201} INFO - Done. Returned value was: Data cuaca berhasil disimpan. 5 rekaman baru, 25 total dalam sejarah
[2025-05-28T10:45:04.442+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_data_etl, task_id=load_weather_task, execution_date=20250528T104000, start_date=20250528T104503, end_date=20250528T104504
[2025-05-28T10:45:04.503+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-28T10:45:04.574+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
